tutorials/attribution/enabling-unified-conversions.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Enabling Conversions within Unified Package
+position: 3
+---
+
+Snowplow’s Attribution package requires the optional conversions module to be configured within the Unified package, because the `snowplow_unified_conversions` table is used as a source by default. This is an additional incremental table that will be updated for each run of the Unified package.
+
+Enabling Conversions will also add 6 extra columns into the sessions table. See more details about that on [this page](https://docs.snowplow.io/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-unified-data-model/conversions/).
+
+If you already have conversion events and the conversion module is enabled, you can skip this step.
+
+:::info 
+Please note: All the actions within this step are to be completed within your Snowplow **Unified Package** dbt project.
+:::
+
+### Configuring Conversions in Unified
+
+1. Add `snowplow__conversion_events: []` variable to dbt_project.yml file inside your project where you the run Snowplow Unified package.
+2. Add a conversion event into the array as shown below to the dbt_project.yml file in the vars section.
+
+You can see further details on configuring the conversion event on [this page of the docs](https://docs.snowplow.io/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-unified-data-model/conversions/) 
+
+:::info 
+Please Note: The total value of conversions must be greater than 0 for the Attribution Package as it is required to calculate the proportion of value brought in by a campaign or channel.
+:::
+
+```yml
+vars:
+  snowplow_unified:
+    snowplow__conversion_events: [
+      {
+        "name": "purchase", // Required, name of the conversion, string (must be valid SQL column name)
+        "condition": " event_name == 'purchase' ", // Required, valid SQL condition that returns true or false
+        "value": " 1 ", // Optional but recommended, can be field name or SQL
+        "default_value": "0", // Optional, can be field name or SQL
+        "list_events": "true"// Optional, boolean
+      }
+    ]
+```
+
+1. Enable the Conversions Module to create the `snowplow_unified_conversions` table by setting the following variable in your Unified dbt_project.yml file. 
```

Comment:
I am being very nitpicky! This should be step 3

Reply:
Not really, i missed that!